src/tendencies/edmfx_closures.jl

```diff
@@ -134,7 +134,7 @@ function edmfx_entr_detr_tendency!(Yₜ, Y, p, t, colidx, turbconv_model::EDMFX)
             ᶜz[colidx],
             ᶜp[colidx],
             Y.c.ρ[colidx],
-            sfc_conditions[colidx].buoy_flux,
+            sfc_conditions[colidx].buoyancy_flux,
```

Comment:
Could you also change the name in `src/tendencies/edmfx_closures.jl`?

Reply:
This is the only place I could still find the name `buoy_flux`